### PR TITLE
Document scanned CVEs for tap-auth

### PR DESCRIPTION
**Pull request by julia-pu**

### CODE REVIEW

File: authn-authz/sca-scanning-results.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
+# SCA scanning results
+## 1.0.1-beta.1
+
+### OSSPI Binary Scan
+
+* Date: March 28, 2022
+* Results: No known vulnerabilities. See scan results in [OSSPI](https://osspi.eng.vmware.com/audit-reports/24019/oss-packages/all?limit=25&offset=0&audit_id=24019&ordering=id).
```
Comment (Manifaust) [inline authn-authz/sca-scanning-results.md]:
```suggestion
* Results: No known vulnerabilities.
```

Readers won't be able to access eng.vmware.com addresses

File: authn-authz/sca-scanning-results.md [hunk 2]
```diff
@@ -0,0 +1,14 @@
+# SCA scanning results
+## 1.0.1-beta.1
```
Comment (Manifaust) [inline authn-authz/sca-scanning-results.md]:
```suggestion
## `rbac` CLI plugin — 1.0.1-beta.1
```

Make it clear that it's for the CLI plugin, not the TAP package

File: authn-authz/sca-scanning-results.md (hunk 1 above)
Comment (julia-pu) [reply authn-authz/sca-scanning-results.md]: okay removed the OSSPI link

File: authn-authz/sca-scanning-results.md (hunk 2 above)
Comment (julia-pu) [reply authn-authz/sca-scanning-results.md]: Added

**Pull request merged**